modules/javafx.controls/src/main/java/javafx/scene/control/SkinBase.java

```diff
@@ -207,6 +214,17 @@ protected final void consumeMouseEvents(boolean value) {
         }
     }
 
+    /**
+     * Returns the skin's instance of {@link ListenerHelper}, creating it if necessary.
+     *
+     * @since 20
+     */
+    protected ListenerHelper listenerHelper() {
```

Comment:
This would make it part of the public API, so you will need to find some other way to do this without adding anything to the public API surface.

Reply:
I *do* want this as public API.

I can change the code to use an accessor, but it gets really ugly:

```
ListenerHelper.get(PaginationSkin.this).addListChangeListener(getSkinnable().getStyleClass(), (ch) -> {
```

Besides, what's the harm of adding a [better] public API?  I do want ListenerHelper to be public anyway.

Reply:
changed the code to remove listenerHelper() to avoid changing the public API.
For now, will be using accessor, and using a local variable in skins where we have many calls to LH (in constructors, for example).